README.md

```diff
@@ -1,40 +1,24 @@
-|        map_tf_publish_rate        |                              Rate at which the map tf is published (in Hertz).                              |              (0, ∞)              |                            10.0                            |
-|           max_idle_time           |           Delay to wait being idle before shutting down ROS when is_online is false (in seconds).           |              [0, ∞)              |                            10.0                            |
-|        min_dist_new_point         |        Distance from current map points under which a new point is not added to the map (in meters).        |              [0, ∞)              |                            0.03                            |
-|         sensor_max_range          |                             Maximum reading distance of the laser (in meters).                              |              [0, ∞)              |                            80.0                            |
-|           prior_dynamic           |                                A priori probability of points being dynamic.                                |              [0, 1]              |                            0.6                             |
-|         threshold_dynamic         |                       Probability at which a point is considered permanently dynamic.                       |              [0, 1]              |                            0.9                             |
-|          beam_half_angle          |                     Half angle of the cones formed by the sensor laser beams (in rad).                      |             [0, π/2]             |                            0.01                            |
-|             epsilon_a             |                                 Error proportional to the sensor distance.                                  |              [0, ∞)              |                            0.01                            |
-|             epsilon_d             |                                Fix error on the sensor distance (in meters).                                |              [0, ∞)              |                            0.01                            |
-|               alpha               |                       Probability of staying static given that the point was static.                        |              [0, 1]              |                            0.8                             |
-|               beta                |                      Probability of staying dynamic given that the point was dynamic.                       |              [0, 1]              |                            0.99                            |
-|               is_3D               |                       true when a 3D sensor is used, false when a 2D sensor is used.                        |          {true, false}           |                            true                            |
```

Comment:
Shouldn't this parameter's documentation stay here?

Reply:
Idk what exact parameters you are referring to, but they were moved to the mapping_config file and are not node parameters anymore.

Reply:
I am talking about the is_3D parameter, I don't see it in the mapping_config

Reply:
Oh I see! Yeah, that parameter should be kept; on my side, GitHub showed me all the removed parameters, so it was super confusing. I'll put it back